es6/lib/utils.js

```diff
@@ -272,7 +272,7 @@ var range = function(max) {
 };
 
 var hasKey = function(obj, key) {
-  return typeof obj[key] !== undefined;
+  return typeof obj[key] !== 'undefined';
```

Comment:
agree with @TimBeyer here. If you want to set an assignment value as having no value you should use null instead of undefined. Undefined is intended to represent the case where no assignment was made to the variable, whereas null is intended to represent the case where you want to explicitly assign no value to the variable.